gradle.properties

```diff
@@ -25,3 +25,6 @@ org.gradle.parallel=true
 # The developer environment is optimized for speed and feedback so we nearly always run Gradle jobs with the daemon.
 android.enableBuildCache=true
 org.gradle.daemon=true
+android.useAndroidX=true
+android.enableJetifier=true
+android.jetifier.blacklist = butterknife-compiler   
```

Comment:
Use latest butterknife

Reply:
Also, add a newline at the last line

Reply:
done. by the way, what is the love for adding a line?

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline